circle.yml

```diff
@@ -546,6 +546,20 @@ jobs:
           min_time: "0.01"
       - spectest
 
+  m1-native:
+    machine:
+      resource_class: macos.m1.large.gen1
```

Comment:
This task failed: https://app.circleci.com/pipelines/github/wasmx/fizzy/7204/workflows/2386fdfd-78ad-4576-b2b0-781280e6b41b/jobs/67260

> Job was rejected because resource class macos.m1.large.gen1, image default is not a valid resource class

According to https://circleci.com/docs/using-macos/#available-resource-classes unfortunately we can't use these:
> The macos.x86.metal.gen1 resource requires a minimum 24-hour lease. See the [Dedicated Host for macOS](https://circleci.com/docs/dedicated-hosts-macos/) page to learn more about this resource class.
>
> The large and macos.m1.large.gen1 resource classes are only available for customers with an annual contract. [Open a support ticket](https://support.circleci.com/hc/en-us/requests/new) if you would like to learn more about our annual plans.